src/Shared/HandyControl_Shared/Controls/Carousel/Carousel.cs

```diff
@@ -190,8 +190,13 @@ public int PageIndex
             else
                 _pageIndex = value;
             UpdatePageButtons(_pageIndex);
+            PageIndexChanged?.Invoke(this, _pageIndex);
         }
     }
+    /// <summary>
+    ///     页码改变事件
+    /// </summary>
+    public EventHandler<int> PageIndexChanged;
```

Comment:
既然是事件了，那就加个 event 关键字吧


```suggestion
    public event EventHandler<int> PageIndexChanged;
```